.travis.yml

```diff
@@ -5,7 +5,8 @@ node_js:
 script:
   - npm run jsHint -- --failTaskOnError
   - npm run test -- --failTaskOnError --suppressPassed
+
 after_success:
-## We only need to run coveralls for one node version (doesn't matter which one).
-## We also ignore publishing failures, since restarting an existing travis build would otherwise break.
-  - if node --version | grep -q ^v6 ; npm run coveralls ; fi
+  ## We only need to run coveralls for one node version (doesn't matter which one).
+  ## We also ignore publishing failures, since restarting an existing travis build would otherwise break.
+  - if [$(node --version) == v6*]; then npm run coverage && npm run coveralls; fi
```

Comment:
Not sure why this was changed, but it broke travis publishing and coveralls.  See https://travis-ci.org/AnalyticalGraphicsInc/obj2gltf/jobs/214002108#L701